spec/Section 4 -- Introspection.md

```diff
@@ -319,7 +319,7 @@ Fields
 
 #### Input Object
 
-Input objects are composite types used as inputs into operations defined as a
+Input objects are composite types used as inputs to operations defined as a
 list of named input values.
```

Comment:
Perhaps:

```
Input objects are composite types used as inputs to some arguments and variables, they are defined as a list of named input values.
```

Riffing off of the previous text `GraphQL describes the types of data expected by arguments and variables.`

Reply:
This has been addressed.